CatFoodCalc/cat_calc.py

```diff
@@ -1,43 +1,36 @@
-class Cat:
-    # age = {
```

Comment:
Unused code should be removed, not commented out. Worst case scenario, write a function prototype that is not executed anywhere.